page-controller/README.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: pattern
+title: Page Controller
+folder: page-controller
+permalink: /patterns/page-controller/
+categories: Structural
+language: en
+tags:
+- Decoupling
+---
```

Comment:
The YAML frontmatter should conform to the instructions here https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute